app/code/Magento/Sales/Controller/Adminhtml/Order/AbstractMassAction.php

```diff
@@ -23,23 +23,27 @@
     protected $redirectUrl = '*/*/';
 
     /**
-     * @var \Magento\Ui\Component\MassAction\Filter
+     * @var object
      */
-    protected $filter;
+    protected $collectionFactory;
 
     /**
-     * @var object
+     * @var \Magento\Ui\Component\MassAction\Filter
      */
-    protected $collectionFactory;
+    protected $filter;
 
     /**
      * @param Context $context
      * @param Filter $filter
+     * @param CollectionFactory $collectionFactory
+     * @param OrderManagementInterface $orderManagement
```

Comment:
It seems the only subclass from \AbstractMassAction using the OrderManagmentInterface is \MassHold where it is injected via constructor. I don't see a need to declare it as param here.